driver.go

```diff
@@ -153,7 +153,11 @@ func (c ocConn) Ping(ctx context.Context) (err error) {
 
 	if c.options.Ping && (c.options.AllowRoot || trace.FromContext(ctx) != nil) {
 		var span *trace.Span
-		ctx, span = trace.StartSpan(ctx, "sql:ping",
+		spanName := "sql.ping"
```

Comment:
changed from `sql:ping` to `sql.ping` - we should keep the default name the same.